src/mumble/GlobalShortcut.cpp

```diff
@@ -565,14 +574,6 @@ GlobalShortcutConfig::GlobalShortcutConfig(Settings &st) : ConfigWidget(st) {
 
 	qcbEnableGlobalShortcuts->setVisible(canDisable);
 
-	qlWaylandNote->setVisible(false);
-#ifdef Q_OS_LINUX
-	if (EnvUtils::waylandIsUsed()) {
-		// Our global shortcut system doesn't work properly with Wayland
-		qlWaylandNote->setVisible(true);
-	}
-#endif
```

Comment:
If I understand things correctly, the new system only works, if DBus support is enabled. Therefore, I think we probably want to keep showing this note in cases in which Mumble is built without DBus.
(In case Mumble is built with DBus, the `USE_DBUS` macro will be defined)

Or maybe even better: show the note, if `!GlobalShortcutXdp::isAvailable()`

Reply:
This is not about the distro having no DBus support. But (for whatever reason) we have an option to build Mumble without DBus support.
I'll check with the rest of the team whether we can simply remove that option though. If we do, then there is no need for you to make these kinds of changes... I'll keep you posted

Reply:
Alright, we decided to drop the dbus option. Thus, you can simply assume to always have DBus available :+1: